ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -0,0 +1,371 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@objc(MSFBottomSheetControllerDelegate)
+public protocol BottomSheetControllerDelegate: AnyObject {
+    /// Called after the sheet fully expanded.
+    @objc optional func bottomSheetControllerDidExpand(_ controller: BottomSheetController)
+
+    /// Called after the sheet fully collapsed.
+    @objc optional func bottomSheetControllerDidCollapse(_ controller: BottomSheetController)
+}
+
+@objc(MSFBottomSheetController)
+public class BottomSheetController: UIViewController {
+
+    /// Initializes the bottom sheet controller.
+    /// - Parameter contentViewController: The view controller that's placed inside the bottom sheet.
+    ///
+    /// By default the root view of `contentViewController` will be sized automatically to fill the available area,
+    /// respecting the provided `preferredExpandedHeightFraction` multiplier.
+    /// Alternatively, the content can size itself by setting `respectsPreferredContentSize` to true and providing a `preferredContentSize`.
+    @objc public init(with contentViewController: UIViewController) {
+        self.contentViewController = contentViewController
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    /// A scroll view in `contentViewController`'s view hierarchy.
+    /// Provide this to ensure the bottom sheet pan gesture recognizer coordinates with the scroll view to enable scrolling based on current bottom sheet position and content offset.
+    @objc open var hostedScrollView: UIScrollView?
+
+    /// Indicates if the bottom sheet is expandable.
+    @objc open var isExpandable: Bool = true {
+        didSet {
+            if isExpandable != oldValue {
+                resizingHandleView.isHidden = !isExpandable
+                panGestureRecognizer.isEnabled = isExpandable
+                move(to: .collapsed, animated: false)
```

Comment:
we need to make sure once VO double tap on the handle and the animation is done for resizing, the VO focus on the object correctly

Reply:
Which object should it focus on and can I somehow test this in simulator?

Reply:
I think the current DrawerController behavior is that to stay in the resize handle. no there isn't a good way to test VO focus on simulator you need to do it in device

Reply:
Can confirm focus stays on the focus stays on the resize handle.